react/features/base/util/interceptComponent.js

```diff
@@ -26,6 +32,17 @@ const _RULES = [
         if (OS === 'android' || OS === 'ios') {
             return UnsupportedMobileBrowser;
         }
+    },
+    () => {
+        if (APP.unsupportedBrowser) {
+            const { isOldBrowser } = APP.unsupportedBrowser;
+
+            if (isOldBrowser) {
+                return UnsupportedDesktopBrowser;
+            }
+
+            return PluginRequiredBrowser;
```

Comment:
The main idea of rules in interceptComponent.js is to describe cases when we should intercept normal "flow" of the app (in this case normal flow is when user opens the app in non-old and non-plugin required browser). So we're showing this case by default and override it by rules if required. Do you see what I mean?